### PR TITLE
Align code per rubocop specs

### PR DESCRIPTION
**Pull request by mgrebenets**
Fix code alignment with help of rubocop. 
Also replace tabs with spaces where occurred.

I couldn't fix the alignment of multiline arrays in `crashlytics_spec.rb` yet, will try to do it in another PR.

### CODE REVIEW

File: lib/fastlane/actions/slack.rb [hunk 1]
```diff
@@ -27,11 +27,11 @@ def self.run(options)
 
         notifier = Slack::Notifier.new(options[:slack_url])
 
-        if options[:username].to_s.length > 0
-          notifier.username = options[:username]
-        else
-          notifier.username = 'fastlane'
-        end
+        notifier.username = if options[:username].to_s.length > 0
```
Comment (KrauseFx) [inline lib/fastlane/actions/slack.rb]: I don't like having an `if` next to the `=`. Could you update the `rubocop` rule to not do that any more and maybe even discourage it?

Comment (mgrebenets) [inline lib/fastlane/actions/slack.rb]: Checking for this rule seems to be new to `rubocop` `0.36.0`. I'll look into disabling and discouraging this rule in `0.36.0` branch, in this PR I just discarded all changes like `var = if ...`, all that's left is fixing alignment and spaces issues.

**Comment by KrauseFx** [comment]
We can close this now, since we locked rubocop, right?

**Comment by mgrebenets** [comment]
Right, I'm tracking `rubocop` `0.36.0` related changes here: https://github.com/mgrebenets/fastlane/tree/rubocop-0.36.0

**Pull request closed without merging**